src/components/DeploymentScheduleCard.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <p-list-item class="schedule-card">
+    <p-tooltip :text="schedule.schedule?.toString({ verbose: true })">
+      <div class="schedule-card__content">
+        {{ schedule.schedule?.toString({ verbose: false }) }}
+      </div>
+    </p-tooltip>
+    <div class="schedule-card__action">
+      <DeploymentScheduleToggle :deployment="deployment" :schedule="deploymentSchedule" @update="updateActiveStatus" />
+      <DeploymentScheduleMenu
+        class="deployment-schedule__menu"
+        size="xs"
+        show-all
```

Comment:
Replace `size="xs"` with just `small`. I don't think `show-all` does anything here so can probably just remove that.